deploy/githooked.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+var githooked = require("githooked"),
+    program   = require("commander");
+
+program
+  .option("-b, --branch <branch>", "Git branch to use when waiting for posthooks")
+  .option("-c, --command <command>", "Command to run after posthook.")
+  .option("-p, --port <number>", "Port in which to watch for postcommit web hooks.")
+  .parse(process.argv);
+
+if (!program.branch || !program.command || !program.port) {
+  console.error("--branch, --command, and --port are all required.");
+  program.outputHelp();
+  process.exit(1);
+}
+
+// Passed through to bodyParser.json().
+// // https://www.npmjs.com/package/body-parser#limit
+var json_options = { limit: 1 << 20 };
```

Comment:
Identifier 'json_options' is not in camel case.<br>Unexpected use of '<<'.<br>'json_options' is defined but never used.